.gitignore

```diff
@@ -5,3 +5,9 @@ dist
 lib
 es
 coverage
+
+website/translated_docs
+website/build/
+website/yarn.lock
```

Comment:
Please don't use Yarn. We use npm here. Also, don't ignore the lock file generated.

Reply:
Noted